TopAnalysis/python/postprocessing/fcncKinematicReco.py

```diff
@@ -0,0 +1,181 @@
+import ROOT
+import math
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+
+import os
+from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection
+from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
+
+class FCNCKinematicReco(Module, object):
+    def __init__(self, *args, **kwargs):
+        pass
+    def beginJob(self):
+        pass
+    def endJob(self):
+        pass
+    def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
+        self.out = wrappedOutputTree
+        self.out.branch("SMTop_pt", "F")
+        self.out.branch("SMTop_eta", "F")
+        self.out.branch("SMTop_phi", "F")
+        self.out.branch("SMTop_mass", "F")
+        self.out.branch("FCNCTop_pt", "F")
+        self.out.branch("FCNCTop_eta", "F")
+        self.out.branch("FCNCTop_phi", "F")
+        self.out.branch("FCNCTop_mass", "F")
+        self.out.branch("Reconstructed", "i")
```

Comment:
I prefer
"KinTopWb_*"
"KinTopZq_*"
"KinTop_status"

Reply:
This comment is applied on code.